app/graphql/types/interfaces/rules_preload.rb

```diff
@@ -40,6 +40,17 @@ def all_rules
         end
       end
 
+      def failing_rules(args = {})
+        ids = ::RuleResult.latest(args[:policy_id]).failed
+                          .group(:rule_id)
+                          .select(:rule_id, 'COUNT(result) as cnt')
+                          .order('COUNT(result) DESC')
```

Comment:
Is the order in this subquery important? Does it have an effect within the `INNER JOIN`?

Reply:
Nice catch, the other `order` call sorts this out. But yes, frontend deals with this better if it's sorted...